runner.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
```

Comment:
script name could be more explicit. is it meant to autorun, or just wrap the pipeline. and what exactly does it run?

Reply:
I agree. It is the script needs to run the nextflow directly using the uploaded files after checking the meta file.

Reply:
removed from the repo.